packages/connect-analytics/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@trezor/connect-analytics",
-    "version": "1.0.16-beta.1",
+    "version": "1.1.0",
```

Comment:
yes, bumping a version but not having any changes is justified by going from beta to stable. ok

Reply:
Yes, that is part of the script on purpose.